sample-apps/react/react-dogfood/components/InvitePanel/InvitePanel.tsx

```diff
@@ -82,25 +83,61 @@ export const Invite = () => {
 };
 
 export const InvitePanel = () => {
+  const isDemoEnvironment = useIsDemoEnvironment();
+  const [expanded, setExpanded] = useState(false);
+
   const qrCodeContent = new URL(window.location.toString());
   qrCodeContent.searchParams.set('from_qr', 'true');
   return (
     <div className="rd__invite">
       <Invite />
       <div className="rd__invite__qr">
-        <h2 className="rd__invite__qr-header">Test on mobile</h2>
-        <p className="rd__invite__qr-description">
-          To test on a mobile device, scan the QR Code below:
-        </p>
-        <div
-          className="rd__invite__qr-container"
-          title={qrCodeContent.toString()}
-        >
-          <QRCodeSVG
-            className="rd__invite__qr-code"
-            value={qrCodeContent.toString()}
-          />
-        </div>
+        {isDemoEnvironment ? (
```

Comment:
```suggestion
        {!isDemoEnvironment ? (
```

This condition should be flipped. QR code should always be visible on our demo environment, collapsed everywhere else.